share/models/logs.py

```diff
@@ -107,6 +108,10 @@ class Meta:
         abstract = True
         ordering = ('-date_modified', )
 
+    @property
+    def status_identifier(self):
+        return next(k for k, v in type(self).STATUS._identifier_map.items() if v == self.status)
```

Comment:
Seems like the color map should use `AbstractBaseLog.STATUS.created`, etc as keys. A bit more verbose but less brittle.